niceman/config.py

```diff
@@ -31,14 +40,8 @@ class ConfigManager(SafeConfigParserWithIncludes, object):
 
     Configuration files (INI syntax) in multiple location are parsed
     when a class instance is created or whenever `Config.reload()` is
-    called later on.  Files are read and parsed in the following
-    order:
-
-    1. '/etc/niceman/niceman.cfg'
-    2. 'niceman/config' in all directories defined by $XDG_CONFIG_DIRS
-       (by default: /etc/xdg/)
-    3. 'niceman.cfg' in $XDG_CONFIG_HOME (by default: ~/.config/)
-    4. 'niceman.cfg' in the current directory
+    called later on.  Files are read and parsed in the order described by
+    `LOCATIONS_DOC`.
```

Comment:
well, you could embed the value here.  We do similar in https://github.com/datalad/datalad/blob/master/datalad/config.py#L29

Reply:
I don't think that's true.  Did you note my attempt to do that in 1e4c207 and then the followup 6c89d18?  I think the core issue is that, in py2, the `__doc__` of a class isn't writeable (AFAICS datalad's _where_reload_doc is only decorating functions).

Am I missing something?

Reply:
Jesus... forgot about that:  I was certain that it should work since in PyMVPA we do compile `__doc__` for "classes" on the fly.  But looking at the code I realized that we are either doing it "per instance" or within metaclass (apparently could do it there, for the record - `AttributesCollector`).  So - my bad

Reply:
> Jesus... forgot about that

I was totally unaware of the issue until I saw the py2 failures in this PR :]